test/README.md

```diff
@@ -37,3 +37,34 @@ The test harness runs a specified CLI  binary(either "../fn" or "TEST_CLI_BINARY
 * Don't be spammy : You shouldn't log excessively in tests as this will impact diagnosability when a test fails.  Instead, always log the `CmdResult` you got from the last command that failed - this should include enough diagnostic history to work out what went wrong (including previous commands)
 * Write parallelizable tests: Tests are slow so sequencing them will make the test package slow - the harness includes tools to help make tests isolated (e.g. any app names created with `h.NewFuncName()` will be deleted after a test is done )  - remember to defer `h.Cleanup()` to ensure test state is cleaned up
 * Watch out for the Environment: The CLI package will pass on the surrounding environment to the CLI when its called - (primarily to allow easily overriding local  configuration like FN_API_URL and other env vars)
+
+## Testing against OCI APIs ##
```

Comment:
Could we test this more tightly, with more mocking rather than relying on having to make actual calls?

Reply:
The current integration tests provide end to end tests against real production OCI API endpoints. This should provide more than sufficient test coverage for this change.

Reply:
Yeah, my only concern here is that we can't automate that through CI easily